docs/tutorials/development-setup.mdx

```diff
@@ -203,108 +114,73 @@ Finally, manually install the following:
 </TabItem>
 </Tabs>
 
-## Stripe
```

Comment:
I moved all the Stripe/Secret stuff to subplat's setup (where there was already a much more complete list of config options)